In [16]:
%matplotlib inline


import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns

# Configure libraries
# The seaborn library makes plots look nicer
sns.set()
sns.set_context('talk')

# Don't display too many rows/cols of DataFrames
pd.options.display.max_rows = 7
pd.options.display.max_columns = 8

# Round decimals when displaying DataFrames
pd.set_option('precision', 2)

In [28]:
df = pd.read_csv("designer.csv")
df = df[['Job Title', 'Salary Estimate', 'Industry', 'Sector','Location']]
df['Location'] = df['Location'].str[-1:-3]
df

Job Title               Salary Estimate  \
0                        Product Designer  $129K-$153K (Glassdoor est.)   
1         Product Designer: Subscriptions  $129K-$153K (Glassdoor est.)   
2          Fire Sprinkler System Designer  $129K-$153K (Glassdoor est.)   
..                                    ...                           ...   
997      Senior Product Design/UX Manager   $94K-$187K (Glassdoor est.)   
998       Senior User Experience Designer   $94K-$187K (Glassdoor est.)   
999  Experience Designer (UX, UI, Visual)   $94K-$187K (Glassdoor est.)   

                                    Industry  \
0                                   Internet   
1                                 Publishing   
2                               Construction   
..                                       ...   
997       Home Furniture & Housewares Stores   
998  Enterprise Software & Network Solutions   
999                              IT Services   

                                 Sector Location  
0                Information Technology           
1                                 Media           
2    Construction, Repair & Maintenance           
..                                  ...      ...  
997                              Retail           
998              Information Technology           
999              Information Technology           

[1000 rows x 5 columns]

In [15]:
location = df.Location.value_counts()
print(location.to_string())

San Francisco, CA                    110
New York, NY                          82
Los Angeles, CA                       63
Menlo Park, CA                        45
Denver, CO                            41
Chicago, IL                           41
Saint Louis, MO                       28
Boston, MA                            26
Houston, TX                           24
Dallas, TX                            24
San Jose, CA                          23
Philadelphia, PA                      23
Mountain View, CA                     22
Columbia, MD                          21
United States                         21
McLean, VA                            20
Framingham, MA                        20
Carson City, NV                       19
Rogers, AR                            19
Boxborough, MA                        19
East Hampton, NY                      19
Kansas City, MO                       19
Washington, DC                         9
Austin, TX                             8
Seattle, WA     

It is shown that SF has 110 of the listings, New York has 82 and LA has 63 of the listings at 